tests/WorkOS/DirectorySyncTest.php

```diff
@@ -8,7 +8,7 @@ class DirectorySyncTest extends \PHPUnit\Framework\TestCase
         setUp as traitSetUp;
     }
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
I'm guessing this is the reason we had to drop support for PHP 7.0?

Not sure if dropping support for an older version qualifies as a breaking change or not 🤔 

Might be something we have to think about before we do the next release.

Reply:
Thanks Marshall!